src/main/scala/cognite/spark/v1/FlexibleDataModelRelationUtils.scala

```diff
@@ -671,7 +667,12 @@ object FlexibleDataModelRelationUtils {
               .leftMap(e =>
                 new CdfSparkException(
                   s"Error parsing value of field '$propertyName' as a json object: ${e.getMessage}"))
-
+          case _: TimeSeriesReference =>
+            Try(InstancePropertyValue.TimeSeriesReference(String.valueOf(row.get(i)))).toEither
+//          case _: FileReference =>
+//            Try(InstancePropertyValue.FileReference(String.valueOf(row.get(i)))).toEither
+//          case _: SequenceReference =>
+//            Try(InstancePropertyValue.SequenceReference(String.valueOf(row.get(i)))).toEither
```

Comment:
```suggestion
```

Reply:
These are needed actually.
FDM has their PRs pending to support these types.
As soon as they're merged, we need to release these.
So I kept them as commented.